### ISSUE TITLE
expose attribute with name method does not work correctly

### ISSUE THREAD

**Issue by suddani** [ruby-grape/grape-entity#218]
I have a JsonRPC object that looks like 
```ruby
{
 :method => String,
 :id => Integer,
 :jsonrpc => "2.0",
 :params => Hash
}
```
When i write an entity to expose those fields like
```ruby
class Entity < Grape::Entity
      expose :method
      expose :id
      expose :jsonrpc
      expose :params
end
```
This does not work and throws `wrong number of arguments (given 0, expected 1) ` for **method**
To work around this i have to write 
```ruby
class Entity < Grape::Entity
      # are you kidding me grape? i can't name my attribute method? really?
      expose :method do |instance,options|
        instance.method
      end
      expose :id
      expose :jsonrpc
      expose :params
end
```

**Comment by Linell** [comment]
That should've actually been fixed by #217

**Comment by dblock** [comment]
Right on, I'll close this, @suddani please confirm that this is fixed or reopen the issue.
